extensions/Needinfo/Extension.pm

```diff
@@ -25,6 +25,7 @@ BEGIN {
 }
 
 sub _user_needinfo_blocked {
+    return 1 if $_[0]->login =~ /\@users\.noreply\.github\.com$/;
```

Comment:
this is a weird bit of hardcoding, especially without comment.

it should probably be a hook.

Reply:
a comment, yes, but this is a hot code path.